arquillian/gradle-adapter/src/main/java/org/wildfly/swarm/arquillian/adapter/gradle/GradleDependencyAdapter.java

```diff
@@ -15,20 +15,29 @@
  */
 package org.wildfly.swarm.arquillian.adapter.gradle;
 
+import java.io.File;
```

Comment:
Please review the [original file](https://github.com/roguexz/thorntail/blob/master/arquillian/gradle-adapter/src/main/java/org/wildfly/swarm/arquillian/adapter/gradle/GradleDependencyAdapter.java) to view the details of the approach better.